libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -299,7 +299,12 @@ void AP_BoardConfig::px4_setup_drivers(void)
 
     if (px4.board_type == PX4_BOARD_PH2SLIM ||
         px4.board_type == PX4_BOARD_PIXHAWK2) {
-        _imu_target_temperature.set_default(60);
+        _imu_target_temperature.set_default(45);
```

Comment:
It takes to long to reach 60C for most users, such that the desired effects/behavior of the cube aren't being reached. The target IMU temp is just meant to be a stable temperature that is above what the cube would heat up to without any assistance normally. Only in really warm climates would you need to set 60.